SeaPublicWebsite/Controllers/FeedbackController.cs

```diff
@@ -34,6 +35,39 @@ public IActionResult FeedbackForm_Post(FeedbackFormViewModel viewModel)
             viewModel.WhatUserToldUs);
         return RedirectToAction(nameof(FeedbackThankYou_Get), "Feedback");
     }
+    
+    [HttpGet("feedback-survey")]
+    public IActionResult FeedbackSurvey_Get()
+    {
+        var viewModel = new FeedbackSurveyViewModel();
+        ViewBag.OnFeedbackPage = true;
+        return View("FeedbackSurvey", viewModel);
+    }
+
+    [HttpPost("feedback-survey")]
+    public IActionResult FeedbackSurvey_Post(FeedbackSurveyViewModel viewModel)
+    {
+        if (!viewModel.VisitReasonList.Any())
+        {
+            ModelState.AddModelError(nameof(viewModel.VisitReasonList), "Please select at least one option");
```

Comment:
I don't think these error messages match the Gov UK style (see https://design-system.service.gov.uk/components/checkboxes/)
e.g. this one should be something like "Select at least one reason that you came to the site today"